packages/compass-aggregations/src/modules/pipeline-builder/builder-helpers.ts

```diff
@@ -22,7 +23,7 @@ export function getStagesFromBuilderState(
   pipelineBuilder: PipelineBuilder
 ) {
   if (state.pipelineBuilder.pipelineMode === 'builder-ui') {
-    return pipelineBuilder.stages.map((stage) => stage.toBSON());
+    return pipelineBuilder.stages.map((stage) => stage.toBSON() as Document);
```

Comment:
Why are you asserting here? Return type of toBSON is Document or null, not just Document

Reply:
cleaned